README.md

```diff
@@ -363,6 +363,17 @@ Alternatively, you can initialize submodules inside the cloned folder:
 $ git submodule update --init --recursive
 ```
 
+### Publishing
+
+1. Increment the version: `npm version ..`
+2. Push to GitHub: `git push --follow-tags`
+3. Wait for Travis and AppVeyor builds to complete
+4. Download prebuilds into `./prebuilds`: `npm run download-prebuilds`
+5. Optionally verify loading a prebuild: `rimraf build && npm t`
+6. Optionally verify which files npm will include: `irish-pub`
+7. Add changelog to the GitHub release
+8. Finally: `npm publish`
```

Comment:
Maybe later we could even have a `prepublish` script (which is run during `npm publish`), which:

1. downloads binaries with `prebuildify-ci`
1. verifies with `rimraf build && npm t`

If either fail then `npm publish` fails.

Reply:
Downside of prepublish script is that it also runs on `irish-pub` IIRC

Reply:
Or rather, that's a downside of `irish-pub` ;)

Reply:
=> #565